agent/agent.go

```diff
@@ -123,24 +119,8 @@ func (a *Agent) handleRequest(ctx *spcontext.Context, id string, msg *privatevcs
 		}
 	}
 
-	ctx = ctx.With(
```

Comment:
I think this with shouldn't be removed, as it's not replicated in the new matcher (nor should it be).

Reply:
Correct, I removed a bit too much.